ioctl/newcmd/config/config.go

```diff
@@ -128,6 +131,78 @@ func (c *info) reset() error {
 	return nil
 }
 
+// set sets config variable
+func (c *info) set(args []string, insecure bool) (string, error) {
+	switch args[0] {
+	case "endpoint":
```

Comment:
the var is used in get and set function.  const  var is better, for hand write by mistake